test/parallel/test-fs-truncate.js

```diff
@@ -239,6 +239,34 @@ function testFtruncate(cb) {
   }));
 }
 
+{
+  const file7 = path.resolve(tmp, 'truncate-file-7.txt');
+  fs.writeFileSync(file7, 'Hi');
+  fs.truncate(file7, undefined, common.mustCall(function(err) {
+    assert.ifError(err);
+    assert(fs.readFileSync(file7).equals(Buffer.from('')));
+  }));
+}
+
+{
+  const file8 = path.resolve(tmp, 'truncate-file-8.txt');
+  fs.truncate(file8, 0, common.mustCall(function(err) {
+    assert.ok(err);
```

Comment:
If we are expecting an error, its bettor to explicitly assert the error. But, I think you meant to write `assert.ifError` here.

Reply:
Since this passes CI, I'm assuming it fails with code `ENOENT`.
Assertion should have been near https://github.com/nodejs/node/blob/9f7e3a404096db60438f6dc305b0a5fa4ecddae7/test/parallel/test-fs-error-messages.js#L521-L545
But since it's not, I agree it should be here.